cmd/execution_builder.go

```diff
@@ -286,7 +286,15 @@ func (exeNode *ExecutionNode) LoadGCPBlockDataUploader(
 			exeNode.collector,
 		)
 
-		bs, err := node.Network.RegisterBlobService(channels.ExecutionDataService, exeNode.executionDataDatastore)
+		bs, err := node.Network.RegisterBlobService(
```

Comment:
I missed this before. Didn't realize we're running 2 separate blob services. I'll open a PR to remove the duplicate.

Can you remove this change to prevent a conflict?

Reply:
PR to remove the duplicate: https://github.com/onflow/flow-go/pull/3418